### PR TITLE
Ot147 83

### PR DESCRIPTION
**Pull request by Lucasfranco1**
**SUMARY**

- Adding Pagination of 10
- Added PagesDto
- Added method list in TestimonialMapper
- Added method for pagination in TestimonialService
- Added path in SecurityConfig for user
- Added getPagesTestimonial in TestimonialController

### CODE REVIEW

File: src/main/java/com/alkemy/ong/controller/TestimonialController.java [hunk 1]
```diff
@@ -32,4 +34,10 @@
         this.testimonialService.deleteTestimonial(id);
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
+    //Pagination, role USER
+    @GetMapping
+    public ResponseEntity<?> getPage(@RequestParam Integer page) {
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/controller/TestimonialController.java]: we can use an int variable instead? faster, what do you think?

**Comment by pablofrankenn** [comment]
Great code. Good for me.

File: src/main/java/com/alkemy/ong/controller/TestimonialController.java (hunk 1 above)
Comment (Lucasfranco1) [reply src/main/java/com/alkemy/ong/controller/TestimonialController.java]: Done!

File: src/main/java/com/alkemy/ong/service/TestimonialService.java [hunk 2]
```diff
@@ -57,6 +63,29 @@ public void deleteTestimonial(String id) {
              throw new ParamNotFound("The testimony is not found in the database.");
         }
     }
+    //Pagination of 10
+    @Transactional
+    public PagesDto<TestimonialDtoFull> getAllForPages(Integer page) {
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/service/TestimonialService.java]: What about searchPaginatedTestimonials for method name?

Comment (Lucasfranco1) [reply src/main/java/com/alkemy/ong/service/TestimonialService.java]: Done!

File: src/main/java/com/alkemy/ong/service/TestimonialService.java [hunk 3]
```diff
@@ -19,6 +26,9 @@
 
     private final TestimonialMapper testimonialMapper;
 
+    @Value("${url.testimonial}")
```
Comment (pablofrankenn) [inline src/main/java/com/alkemy/ong/service/TestimonialService.java:29]: What's the reason behind make the URL a environment variable?

**Pull request merged**